builder/dispatchers.go

```diff
@@ -47,6 +47,22 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 		return fmt.Errorf("Bad input to ENV, too many args")
 	}
 
+	// TODO/FIXME/NOT USED
```

Comment:
Normally I agree, however, in this case because there are no uses of this new flag stuff in the code yet to use as a guide I figured we needed something as part of the commit so someone can use it for the first one (just in case I get hit by a bus tomorrow :-) ). Once the first flag is added we can remove it.